stubs/tqdm/tqdm/dask.pyi

```diff
@@ -0,0 +1,10 @@
+from _typeshed import Incomplete
+
+from dask.callbacks import Callback  # type: ignore
```

Comment:
It's not currently possible to have third-party `py.typed` libraries as dependencies for typeshed stubs. Your best bet is to do something like this:

```python
from typing import Any
from typing_extensions import TypeAlias
Callback: TypeAlias = Any # actually dark.callbacks.Callback
```

Reply:
Thanks, I was wondering how to get around that!